translate_srt.py

```
@@ -205,26 +200,23 @@ def __init__(
     ):
         self.window_size = window_size
         self.max_requests = max_requests
-        self.requests = []
+        self.request_times = []
         self.batch_count = 0
 
     def can_make_request(self) -> bool:
         now = datetime.now()
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)